File: 2491/yeolyi.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().split('\n');
const arr = Array.from(input[1].split(' ')).map(Number);

let cnt = 1;
let ret = 1;

for (let idx = 1; idx < arr.length; idx++) {
    if (arr[idx - 1] <= arr[idx]) {
        cnt++;
        ret = Math.max(ret, cnt);
    } else {
        ret = Math.max(ret, cnt);
        cnt = 1;
    }
}

cnt = 1;

for (let idx = 1; idx < arr.length; idx++) {
    if (arr[idx - 1] >= arr[idx]) {
        cnt++;
        ret = Math.max(ret, cnt);
    } else {
        ret = Math.max(ret, cnt);
        cnt = 1;
    }
}

console.log(ret);

File: Silver II/2117/yeolyi.py
N = int(input())
M = N // 2

if N % 2 == 0:
    print(M**2 - M)
else:
    print(M**2)


# 1 2 3 4 5 6 7

# 1 2 3 4 5 7 6
# 1 2 3 4 7 6 5
# 1 2 3 7 6 5 4
# 3 1 2 7 6 5 4
# 3 2 1 7 6 5 4

# 1 + 2 + 3 + 2 + 1

# 1 2 3 4 5 6

# 1 2 3 4 6 5
# 1 2 3 6 5 4
# 3 1 2 6 5 4
# 3 2 1 6 5 4

# 1 + 2 + 2 + 1

# 1 2 3 4 5
# 1 2 3 5 4
# 1 2 5 4 3

# 1 + 2 + 1

# 1 2 3 4
# 1 2 4 3
# 2 1 4 3

# 1 + 1

# 1 2 3
# 1 3 2

# 1
